src/components/SkillPage/SkillListing.js

```diff
@@ -265,9 +266,12 @@ class SkillListing extends Component {
   };
 
   saveSkillUsage = (skill_usage = []) => {
+    // eslint-disable-next-line
```

Comment:
Needed else it throws a warning of no return statement in arrow function

Reply:
@anshumanv I think you can pass the eslint by putting a `return null` statement at the end.

Reply:
Yes, it passes that way but null data is not to be returned into the data array.